test-it/karma/fixtures/javascriptinvoke/DataTypes.via

```diff
@@ -27,6 +27,14 @@ define(MyVI dv(VirtualInstrument (
         e(dv(.Single 3.0) singleValue)
         e(dv(.Double 6.0) doubleValue)
         e(dv(.String "National Instruments") stringValue)
+        e(dv(a(.Int8 *) (1 2 3)) arrayInt8)
```

Comment:
For integer types, please add the Min and Max value in the integer range. For floating point values please do negative, zero, positive values, at this moment I don't have a better idea about testing them